src/disco/containers/DiscoPane.js

```diff
@@ -67,6 +67,7 @@ class DiscoPane extends React.Component {
           </div>
         </header>
         {results.map((item, i) => <Addon {...camelCaseProps(item)} key={i} />)}
+        <a className="cta" href="https://addons.mozilla.org/">{i18n.gettext('See more add-ons!')}</a>
```

Comment:
Definitely up for discussing this in London and coming up with a plan - might be better to file an issue with the details you want to cover - as this buried in a merged PR is likely to get lost.

I'll see about de-nesting in the RLT updates.